Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW2/src/com/epam/cdp/chef/salad/Salad.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.cdp.chef.salad;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.epam.cdp.chef.interfaces.ICalculateCalories;
+import com.epam.cdp.chef.vegetables.Vegetable;
+
+public class Salad implements ICalculateCalories {
+
+	private List<Vegetable> ingredients = new ArrayList<Vegetable>();
+
+	/**
+	 * @return list of ingredients
+	 */
+	public List<Vegetable> getIngredients() {
+		return ingredients;
+	}
+
+	/**
+	 * Adding vegetable to the salad
+	 * 
+	 * @param veg
+	 */
+	public void add(Vegetable veg) {
+		ingredients.add(veg);
+	}
+
+	/**
+	 * Searching vegetable by name
+	 * 
+	 * @param name
+	 *            of vegetable in salad
+	 * @return vegetable or null in case no matches
+	 */
+	public Vegetable find(String name) {
```

Comment:
I think find methods should return List. not only first element
please use new features

```
return ingredients.stream().filter(vegetable -> vegetable.calculateFullCalories() == calories).collect(
				Collectors.toList());
```